plugin/v2/plugin.go

```diff
@@ -205,18 +213,53 @@ func (p *Plugin) GetTypes() []types.PluginInterfaceType {
 
 // InitSpec creates an OCI spec from the plugin's config.
 func (p *Plugin) InitSpec(s specs.Spec, libRoot string) (*specs.Spec, error) {
-	rootfs := filepath.Join(libRoot, p.PluginObj.ID, "rootfs")
+	p.Rootfs = filepath.Join(libRoot, p.PluginObj.ID, "rootfs")
```

Comment:
p.Rootfs can also be used in https://github.com/docker/docker/blob/master/plugin/backend.go#L132